people_admin/views.py

```diff
@@ -54,14 +54,36 @@ def jurisdiction_list(request):
     )
 
     for state in states:
```

Comment:
for now try changing this to 

for abbr in states + [us.unitedstatesofamerica]:

(add "import us" at top of file)

a broader decision about whether or not it is safe to add us.unitedstatesofamerica to 'states' can be made later, but this would be how it'd work

Reply:
Think I merged it in right but lmk. Left it as `for states` so could access names & abbr